cli/cmd/predict.go

```diff
@@ -98,6 +98,7 @@ var predictCmd = &cobra.Command{
 
 		apiID := predictResponse.ResourceID
 		api := resourcesRes.APIStatuses[apiID]
```

Comment:
nit (and I know this isn't in your change list): `api` here should be called `apiStatus`